documentation/staging/content/release-notes.md

```diff
@@ -6,52 +6,104 @@ draft: false
+| Date               | Version  | Change - See also, [Change log](#change-log).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
+|--------------------|----------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| July 4, 2022       | v4.0.0   | New Cluster resource for HPA. Domain resource "v9" with auxiliary image simplification, improved status reporting, and improved failure retry predictability and transparency. Istio and other service mesh support enabled automatically. Kubernetes 1.24 and 1.25 support. Minimum Kubernetes version is now 1.21.                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
```

Comment:
HPA ->Horizontal Pod Autoscaler (HPA) (should spell out the acronym before using it)